sockeye/pre_mxnet.py

```diff
@@ -34,6 +35,12 @@ def handle_omp_num_threads():
                 val = sys.argv[i + 1]
             logger.warning('Setting %s=%s', OMP_NUM_THREADS, val)
             os.environ[OMP_NUM_THREADS] = val
+        elif arg == ENV_ARG and len(sys.argv) >= i + 1:
+            argval = sys.argv[i + 1]
```

Comment:
Minor comment 1: We could also support the `--arg=val` format for consistency (`--env="OMP_NUM_THREADS=1"`).

Reply:
makes sense, will change